pkg/virt-controller/services/multus_annotations_test.go

```diff
@@ -86,4 +88,73 @@ var _ = Describe("Multus annotations", func() {
 			Expect(multusAnnotationPool.toString()).To(BeIdenticalTo(expectedString))
 		})
 	})
+
+	Context("Generate Multus network selection annotation", func() {
+		var vmi *v1.VirtualMachineInstance
+
+		BeforeEach(func() {
+			vmi = &v1.VirtualMachineInstance{ObjectMeta: metav1.ObjectMeta{Name: "testvmi", Namespace: "default"}}
+			vmi.Spec.Networks = []v1.Network{
+				{Name: "default", NetworkSource: v1.NetworkSource{Pod: &v1.PodNetwork{}}},
+				{Name: "blue", NetworkSource: v1.NetworkSource{Multus: &v1.MultusNetwork{NetworkName: "test1"}}},
+				{Name: "red", NetworkSource: v1.NetworkSource{Multus: &v1.MultusNetwork{NetworkName: "other-namespace/test1"}}},
+			}
+			vmi.Spec.Domain.Devices.Interfaces = []v1.Interface{
+				{Name: "default", Binding: &v1.PluginBinding{Name: "test-binding"}},
+				{Name: "blue"}, {Name: "red"}}
+		})
+
+		When("NetworkBindingPlugins feature enabled", func() {
+			It("should fail if the specified network binding plugin is not registered (specified in Kubevirt config)", func() {
+				kvConfig := &v1.KubeVirtConfiguration{
+					DeveloperConfiguration: &v1.DeveloperConfiguration{
+						FeatureGates: []string{"NetworkBindingPlugins"}}}
+				config, _, _ := testutils.NewFakeClusterConfigUsingKVConfig(kvConfig)
```

Comment:
I think you should create a helper that gets a list of bindings and returns the `config`.

Reply:
Done